### PR TITLE
fix: now when creating a new config with refs, the correct value is sent to the api

### LINKED ISSUE #56: Can't see config references in either config page or new version creation
reproduce:
1. create new config with ref
2. try to create new version of the config, you dont see your original ref, just the resolved value

### PR DESCRIPTION
**Pull request by CptSchnitz**
closes #56

### CODE REVIEW

File: src/pages/createConfig/step2AddConfig/step2AddConfig.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ export const Step2AddConfig: React.FC = () => {
     try {
       const json = JSON.parse(stringValue);
       const { isValid, errors } = await validateJson(schema, json);
```
Comment (netanelC) [inline src/pages/createConfig/step2AddConfig/step2AddConfig.tsx:71]:
```suggestion
      const { isValid, errors } = await validateJson(schema, JSON.parse(stringValue));
```

**Review by netanelC — COMMENTED** [review]
Good Job!

File: src/pages/createConfig/step2AddConfig/step2AddConfig.tsx (hunk 1 above)
Comment (CptSchnitz) [reply src/pages/createConfig/step2AddConfig/step2AddConfig.tsx:71]: 1. this is out of scope of the PR.
2. i think that readability is more important.

Comment (netanelC) [reply src/pages/createConfig/step2AddConfig/step2AddConfig.tsx:71]: OK

**Review by netanelC — APPROVED** [review]

**Pull request merged**